src/pages/Sign/sections/SignUp.tsx

```diff
@@ -102,79 +102,81 @@ const ErrorText = styled('div')({
   fontSize: '16px',
 })
 
+const ErrorMessage = styled('div')({
+  minHeight: '20px',
+  fontSize: '16px',
+})
+
 interface Props {
   title: string
   adress: string
   buttonText: string
   inputTitleEmail: string
   inputTitlePersonalNumber: string
-  errorMessage: string
+  errorMessage?: string
 }
 
-interface State {
-  email: string
-  personalNumber: number
-  signUpError: string
+const validatePersonalNumber = (value: any) => {
```

Comment:
imo we should use `yup` for these type of things. see ie https://github.com/HedvigInsurance/web-onboarding/blob/master/src/pages/Chat/steps/LivingSituationInput.tsx#L36